db-engine/src/main/java/com/splicemachine/db/impl/sql/compile/ScanCostFunction.java

```diff
@@ -190,7 +190,16 @@ private void performQualifierSelectivity (Predicate p, QualifierPhase phase, dou
     }
 
 
-    public void generateOneRowCost() throws StandardException {
+    public void generateOneRowCost(boolean useSpark) throws StandardException {
+        double openLatency = scc.getOpenLatency();
+        double closeLatency = scc.getCloseLatency();
+        double localLatency = scc.getLocalLatency();
+        double remoteLatency = scc.getRemoteLatency();
+        if (!useSpark) {
+            openLatency = openLatency/10;
+            closeLatency = closeLatency/10;
```

Comment:
The open/close latency is pretty conservative compared to the table scan cost, This affects the control path more significantly than spark execution plan. so we want to reduce it.  I didn't change it for the spark path because this change could make nestedloop join and index lookup plan cheaper, which may not be a good choice for Spark queries. It is better to make this change for spark path after we fully separate the costing of control and spark execution.